FastSurferCNN/utils/checkpoint.py

```diff
@@ -12,28 +12,81 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import glob
-
 # IMPORTS
 import os
 from pathlib import Path
-from typing import Collection, Iterable, MutableSequence, Optional, Union
+from typing import MutableSequence, Optional, Union, Any
 
 import requests
 import torch
 import yacs.config
+import yaml
 
 from FastSurferCNN.utils import logging
-from FastSurferCNN.utils.parser_defaults import FASTSURFER_ROOT
+
 
 Scheduler = "torch.optim.lr_scheduler"
 LOGGER = logging.getLogger(__name__)
 
 # Defaults
-URL = "https://b2share.fz-juelich.de/api/files/a423a576-220d-47b0-9e0c-b5b32d45fc59"
-VINN_AXI = FASTSURFER_ROOT / "checkpoints/aparc_vinn_axial_v2.0.0.pkl"
-VINN_COR = FASTSURFER_ROOT / "checkpoints/aparc_vinn_coronal_v2.0.0.pkl"
-VINN_SAG = FASTSURFER_ROOT / "checkpoints/aparc_vinn_sagittal_v2.0.0.pkl"
+from FastSurferCNN.utils.parser_defaults import FASTSURFER_ROOT
+YAML_DEFAULT = FASTSURFER_ROOT / "FastSurferCNN/config/checkpoint_paths.yaml"
+
+
+def get_plane_dict(filename : Path = YAML_DEFAULT) -> dict[str, Union[str, list[str]]]:
+    """
+    Load the plane dictionary from the yaml file.
+
+    Parameters
+    ----------
+    filename : Path
+        path to the yaml file. Either absolute or relative to the FastSurfer root directory.
+    Returns
+    -------
+
+    dict[str, Union[str, list[str]]]
+        dictionary of the yaml file
+
+    """
+    if not filename.absolute():
+        filename = FASTSURFER_ROOT / filename
+
+    with (open(filename, "r")) as file:
+        dictionary = yaml.load(file, Loader=yaml.FullLoader)
+    return dictionary
+
+def get_plane_default(type : str, plane : Optional[str] = None, filename : str | Path = YAML_DEFAULT) -> Union[str, list[str]]:
+    """
+    Get the default value for a specific plane.
+
+    Parameters
+    ----------
+    type : str
+        Type of value. Can be URL, CKPT, CFG
+    plane : str
+        Plane to get the default value for. Can be "axial", "coronal" or "sagittal".
+    filename : str | Path
+        path to the yaml file. Either absolute or relative to the FastSurfer root directory.
```

Comment:
```suggestion
        The path to the yaml file. Either absolute or relative to the FastSurfer root directory.
```

Should start with a capital letter (also no repeating of words).